IOPool/Common/bin/EdmProvDump.cc

```diff
@@ -494,24 +326,43 @@ class ProvenanceDumper {
   edm::ProcessConfigurationVector phc_;
   edm::ProcessHistoryVector phv_;
   ParameterSetMap psm_;
-  HistoryNode historyGraph_;
+  std::vector<HistoryChain> histories_;
   bool showDependencies_;
   bool extendedAncestors_;
   bool extendedDescendants_;
   bool excludeESModules_;
   bool showOtherModules_;
   bool productRegistryPresent_;
   bool showTopLevelPSets_;
+  bool showHistoryID_;
   std::vector<std::string> findMatch_;
   bool dontPrintProducts_;
   std::string dumpPSetID_;
   int const productIDEntry_;
-  bool const printHardwareResourcesDescription_;
+  bool const sort_;
+
+  using ProcessSimpleIDsType = std::map<edm::ProcessConfigurationID, unsigned int>;
```

Comment:
How about a comment about what the unsigned int is and how it is used?

Reply:
Added a comment